android/build.gradle

```diff
@@ -131,8 +131,8 @@ dependencies {
   api 'com.facebook.react:react-native:+'
 
   implementation 'androidx.core:core-ktx:1.8.0'
-  releaseImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk:v3.2.2'
-  debugImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk-debug:v3.2.2'
+  releaseImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk:master-SNAPSHOT'
+  debugImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk-debug:master-SNAPSHOT'
```

Comment:
Just want to confirm that we're targeting SNAPSHOT?  Is this replaced with specific versions when released?

Reply:
Yes, this will be replaced with a specific version before release.